### PR TITLE
Add basic code owners file

### PR DESCRIPTION
**Pull request by asarium**

### CODE REVIEW

**Comment by JohnAFernandez** [comment]
So I had to google what this was for, but now that I have, I see what you're trying to do and approve.

"While effective code review is essential to every successful project, it’s not always clear who should review files—even with GitHub’s reviewer suggestions. Now repository maintainers can define exactly which people and teams need to review projects using code owners. This new feature automatically requests reviews from the code owners when a pull request changes any owned files."

**Comment by Goober5000** [comment]
This makes sense to me.  I'll add a section for myself when it's merged too.

**Review by Goober5000 — APPROVED** [review]

**Pull request merged**
